### DISCUSSION TITLE
Meeting 2024-12-03

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#807] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-12-03)

**Comment by jordens** [comment]
I'd like to nominate https://github.com/rust-embedded/embedded-hal/issues/633#issuecomment-2378909844 for discussion but won't be able to attend. Thanks!

**Reply by adamgreig** [reply]
We discussed this today and the gist was:

1. Encouraging HAL authors to implement core::error in the e-h docs does make sense, we'd welcome a PR updating that
2. It's not as clear if there's a reason to move embedded-hal-error into this repo/crate right now; it's potentially useful but we could wait and see how widely useful it is before doing so; in the meantime it's still available for use in the already-published crate

**Comment by adamgreig** [comment]
[New tracking issue](https://github.com/rust-lang/leadership-council/issues/127) opened for working out where the REWG should fit in the Rust project structure.
